packages/config/src/projects/metis.ts

```diff
@@ -40,9 +40,9 @@ export const metis: Project = {
     provider: 'Optimism',
     riskView: {
       stateValidation: {
-        value: 'In development',
+        value: 'No',
```

Comment:
No is too short IMO.
```suggestion
        value: 'No validation',
```